BRAVE/docker-compose.prod.yml

```diff
@@ -0,0 +1,59 @@
+version: "3.8"
+
+services:
+  django:
+    build: ./backend
+    container_name: dev-django
```

Comment:
if this is meant to be for "production" should the containers be named `prod-django` ?